### ISSUE TITLE
Ubuntu: "Learn More" fails

### ISSUE THREAD

**Issue by uweschaefer** [Mercateo/dwatch#17]
with:

Using PPAPI flash.
[0618/154348:ERROR:nss_util.cc(98)] Failed to create /home/doc/.pki/nssdb directory.
[0618/154348:FATAL:chrome_main_delegate.cc(411)] Check failed: process_type.empty(). Unable to get the user data directory for process type: zygote
#0 0x7f5f3cde739e base::debug::StackTrace::StackTrace()
#1 0x7f5f3ce03e6e logging::LogMessage::~LogMessage()
#2 0x55ee142fbe9b <unknown>
#3 0x7f5f38525c3c <unknown>
#4 0x7f5f385245cd content::ContentMain()
#5 0x55ee142fb29a <unknown>
#6 0x7f5f263f0830 __libc_start_main
#7 0x55ee142fb159 <unknown>

**Comment by otbe** [comment]
@uweschaefer curious :)
Does the github link work for you?

<img width="1435" alt="bildschirmfoto 2016-06-18 um 16 01 04" src="https://cloud.githubusercontent.com/assets/3391052/16171392/1fd4ad64-356e-11e6-916f-86aa6abed7c3.png">

**Comment by uweschaefer** [comment]
nope. other than the above, this one does not blow, but just does not have any effect.

**Comment by otbe** [comment]
Interesting. 
Both should open the github repo in your default browser therefore they execute the same function. One from the main thread (Stacktrace) and one from the render thread (swallowed by JS engine :)) 
Seems to be a linux issue. Will try to fix it tomorrow :)

**Comment by otbe** [comment]
Does ```xdg-open http://github.com``` work for you?
It seems to be a chrome permission issue on your end. Do you run ```DWatch``` as root?

**Comment by uweschaefer** [comment]
xdg-open http://github.com works as non-privilegded user.

and yes, i always run DWatch as root, as otherwise it would not work due to permissions of the .sock file.

Even funnier... If i start DWatch as a normal user, "Learn more" works fine. I guess, it is a chromium profile issue, then. (this root guy does not have the habit of browsing the web :)
